cppcache/include/geode/Cacheable.hpp

```diff
@@ -34,12 +34,6 @@ namespace client {
 
 typedef Serializable Cacheable;
```

Comment:
Is there any point to keeping this file around now?

Reply:
We could totally move this into Serializable.hpp at this point. I'll write up another ticket to investigate the best solution. 

https://issues.apache.org/jira/browse/GEODE-4356